src/main/java/de/rwth/idsg/steve/service/WebUserService.java

```diff
@@ -77,6 +82,10 @@ public void afterStart(ContextRefreshedEvent event) {
         this.createUser(user);
     }
 
+    private void moveApiTokenFromConfigToDatabase() {
+        // TODO
```

Comment:
will implement later

Reply:
my WIP approach/thinking:

check whether some web_user has api_password
- yes: no need to insert anything. do not change existing setups.
- no: assign the api_token as api_password to all web_users with ADMIN authority. (since current default setup has only one user, and that is admin, we will set associate this api_password with it) 

wdyt?

Reply:
I am just thinking. Do you need to migrate the previous token? The change is already breaking with the auth mode change and users will have to adapt the integration. So the effort to create an API password won't be so hard.

Reply:
sure, i would have no problem with this.